example/src/components/ComponentDocTemplate.js

```diff
@@ -50,105 +51,119 @@ const TemplateDescriptionText = styled.p`
+//     margin: 40px 0px 0px 0px;
+//     /* border: 1px solid lightblue; */
+// `
+
+// const InfoWrapper = styled.div`
+//     margin: 0px 0px 0px 0px;
+//     display: flex;
+//     /* border: 1px solid purple; */
+// `
+
+// const InfoBold = styled.p`
+//     margin: 0px 0px 0px 0px;
+//     width: 96px;
+//     font-family: "aktiv-grotesk";
+//     font-style: normal;
+//     font-weight: medium;
+//     font-size: 20px;
+//     line-height: 24px;
+//     /* display: flex; */
+//     /* align-items: center; */
+//     color: #000000;
+//     /* border: 1px dashed gray; */
+// `
+
+// const InfoRegular = styled.p`
+//     margin: 0px 0px 0px 0px;
+//     width: 100%;
+//     max-height: 56px;
+//     /* width: 150px; */
+//     font-family: "aktiv-grotesk";
+//     font-style: italic;
+//     font-weight: lighter;
+//     font-size: 20px;
+//     line-height: 24px;
+//     /* display: flex; */
+//     /* align-items: center; */
+//     color: #000000;
+//     /* border: 1px dashed red; */
+// `
+
+// const InfoDescription = styled.p`
+//     margin: 0px;
+//     font-family: "aktiv-grotesk";
+//     font-style: normal;
+//     font-weight: 300;
+//     font-size: 20px;
+//     line-height: 24px;
+//     display: flex;
+//     align-items: center;
+//     color: #000000;
+//     /* border: 1px dotted black; */
+// `
+
+const HiddenParagraph = styled.p`
+    margin: 0px 0px 0px 8px;
+    height: 24px;
+    display: none;
+    justify-content:center;
     align-items: center;
-    color: #000000;
-    /* border: 1px dotted gray; */
-`
-
-const PropsInfoContainer = styled.div`
-    margin: 40px 0px 0px 0px;
-    /* display: flex; */
-    /* flex-direction: column; */
-    /* border: 1px solid lightblue; */
-`
-
-const InfoWrapper = styled.div`
-    margin: 0px 0px 8px 0px;
-    display: flex;
-    /* border: 1px solid purple; */
-`
-
-const InfoBold = styled.p`
-    margin: 0px 0px 0px 0px;
-    width: 80px;
     font-family: "aktiv-grotesk";
-    font-style: normal;
+    font-style: light;
     font-weight: normal;
-    font-size: 20px;
-    line-height: 24px;
-    display: flex;
-    align-items: center;
+    font-size: 12px;
+    line-height: 16px;
     color: #000000;
-    /* border: 1px dashed gray; */
-`
-
-const InfoRegular = styled.p`
-    margin: 0px 0px 0px 0px;
-    /* width: 150px; */
-    font-family: "aktiv-grotesk";
-    font-style: italic;
-    font-weight: 300;
-    font-size: 20px;
-    line-height: 24px;
-    display: flex;
-    align-items: center;
-    color: #000000;
-    /* border: 1px dashed red; */
-`
-
-const InfoDescription = styled.p`
-    margin: 0px;
-    font-family: "aktiv-grotesk";
-    font-style: normal;
-    font-weight: 300;
-    font-size: 20px;
-    line-height: 24px;
-    display: flex;
-    align-items: center;
-    color: #000000;
-    /* border: 1px dotted black; */
 `
 
 function ComopnentDocTemplate(props) {
     let location = useLocation()
 
     function copyToClipboard() {
```

Comment:
Code smell: this function feels too imperative, but it's 100% possible this is the best way to do this in React. I'll need to do more research. @recondesigns if you have the URL to a site that helped you write this, that could be helpful for me. 👍🏻